Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py

```diff
@@ -0,0 +1,1516 @@
+        if status == 'COMPLETED':
+            scan_id = get_scan_id(account_id, client_id, auth_token, job_id)
+            machine_id = get_scan_id_machine(account_id, client_id, auth_token, job_id)
+            detections = get_scan_detections(account_id, client_id, auth_token, machine_id, scan_id)
+            if len(detections) != 0:
+                remove_empty_detection_fields(detections)
+                return_outputs(
+                    readable_output=tableToMarkdown(f'Scan Detections Report for the Job_Id: {job_id}', detections),
+                    outputs={
+                        'Malwarebytes.Scan(val.Job_ID == obj.Job_ID)': {
+                            'Job_ID': job_id,
+                            'Status': status,
+                            'Detections': detections
+                        }
+                    },
+                    raw_response=detections
+                )
+            else:
+                demisto.results("Scan has been Completed and No Threats have been found !")
+        else:
+            demisto.results('Scan Status for the job_id ' + str(job_id) + ' is ' + str(status))
+
+    else:
+        return_error(message="Please Enter Job_Id", error='')
+
+
+# Function to execute Get Scan Status Command
+def scan_status(account_id, client_id, auth_token, job_id):
+
+    if job_id:
+        status = get_scan_status(account_id, client_id, auth_token, job_id)
+        return_outputs(
+            'Scan Status for the job_id ' + str(job_id) + ' is ' + str(status),
+            outputs={
+                'Malwarebytes.Scan(val.Job_ID == obj.Job_ID)': {
+                    'Job_ID': job_id,
+                    'Status': status
+                }
+            },
+            raw_response=[]
+        )
+
+    else:
+        return_error(message="Please Enter Job_Id", error='')
+
+
+# Function to execute Fetch Incidents Command
+def fetch_incidents(account_id, client_id, auth_token, Event_list):
+
+    if Event_list == 'Suspicious Activity (EPR)':
+
+        suspicious_activities = get_suspicious_activities(account_id, client_id, auth_token)
+
+        # And retrieve it for use later
+        lastRun = demisto.getLastRun()
+        last_fetch_time = lastRun.get("time")
+        last_count = lastRun.get("count")
+        last_custom = lastRun.get("custom")
+        suspicious_activity_severity = demisto.params().get('suspicious_activity_severity')
+        if suspicious_activity_severity:
+            lastPointer, incident = fetch_suspicious_activities(suspicious_activities, suspicious_activity_severity,
+                                                                last_fetch_time, last_count, last_custom)
+            if lastPointer and incident:
+                demisto.incidents([incident])
+                # You can store the last run time...
+                demisto.setLastRun(lastPointer)
+            else:
+                demisto.incidents([])
```

Comment:
lets try to build the incidents list dynamically and call `demisto.incidents()` with it once

Reply:
This is implemented in this way not to overload demisto with alerts. Have been discussed already.